src/Scss/Post/Post.scss

```diff
@@ -1,94 +1,128 @@
 .post {
-    font-family: 'Roboto', sans-serif;
-    margin-top: 1em;
-    margin-right: auto;
-    margin-left: auto;
-    margin-bottom: 1em;
-    display: grid;
-    grid-template-columns: .25fr 1fr;
+  font-family: 'Roboto', sans-serif;
+  margin-top: 1em;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 1em;
+  display: grid;
+  grid-template-columns: .25fr 1fr;
 
-    position: relative;
-    margin-left: 10%;
-    margin-right: 10%;
-    border-radius: 1em;
-    background-color: $application-gray;
-    @extend %drop-shodow;
-    height: fit-content;
-    p {
-        margin-top: 0;
-        padding-top: 0;
-        margin-right: 0.5em;
-    }
+  position: relative;
+  margin-left: 10%;
+  margin-right: 10%;
+  border-radius: 1em;
+  background-color: $application-gray;
+  @extend %drop-shodow;
+  max-height: 20vh;
+  width: 60vw;
+  overflow-y: scroll;
 
-    .post-overlay {
-        width: 100%;
-        height: 100%;
-        position: absolute;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        z-index: 2;
-        background-color: rgba(255,255,255,.6);
+  padding-bottom: .75em;
+  p {
+    margin-top: 0;
+    padding-top: 0;
+    margin-right: 0.5em;
+  }
 
-        img {
-          height: 80%;
-          width: auto;
-          margin: auto;
-          // opacity: 1;
-        }
-    }
+  .post-overlay {
+    width: 100%;
+    // height: 100%;
+    height: auto;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 2;
+    background-color: rgba(255,255,255,.6);
 
-    .post-left {
-        display: grid;
-        grid-template-rows: .5fr .5fr;
-        .post-left-top {
-            display: flex;
-            justify-content: center;
-            #user-icon {
-              @extend %profile-picture;
-              margin-top: 0.5em;
-            }
-        }
-        .post-left-bottom {
-            display: flex;
-            justify-content: center;
-            #like-button {
-                width: 2.7em;
-                margin-top: 0.5em;
-                align-self: center;
-                cursor: pointer;
-            }
-        }
```

Comment:
We do have event listers tied to the like button please check to make sure that the like button does scale in all sizes

Reply:
Maybe! I don't know where this came from, I didn't write / delete this.